src/tests/unit/tests/views/content/__snapshots__/content-panel-button.test.tsx.snap

```diff
@@ -1,43 +1,67 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ContentPanelButton renders from content 1`] = `
-<CustomizedActionButton
-  ariaLabel="info and examples"
-  iconProps={
-    {
-      "iconName": "iconName",
-    }
-  }
-  onClick={[Function]}
->
-  TEXT
-</CustomizedActionButton>
+<DocumentFragment>
+  <mock-customizedactionbutton
+    arialabel="info and examples"
+    iconprops="[object Object]"
+  >
+    TEXT
+  </mock-customizedactionbutton>
+</DocumentFragment>
+`;
+
+exports[`ContentPanelButton renders from content: CustomizedActionButton props 1`] = `
+{
+  "ariaLabel": "info and examples",
+  "children": "TEXT",
+  "iconProps": {
+    "iconName": "iconName",
+  },
+  "onClick": [Function],
+}
 `;
 
 exports[`ContentPanelButton renders from path 1`] = `
-<CustomizedActionButton
-  ariaLabel="info and examples"
-  iconProps={
-    {
-      "iconName": "iconName",
-    }
-  }
-  onClick={[Function]}
->
-  TEXT
-</CustomizedActionButton>
+<DocumentFragment>
+  <mock-customizedactionbutton
+    arialabel="info and examples"
+    iconprops="[object Object]"
```

Comment:
can we make iconprops to be visible by using 
expectMockedComponentPropsToMatchSnapshots(CustomizedActionButton)

Reply:
Not able to implement this.  I have already mocked and used   expectMockedComponentPropsToMatchSnapshots([ActionButton]);  Customized action button gives error.